engine/src/main/java/org/terasology/rendering/dag/nodes/OutputToScreenNode.java

```diff
@@ -59,4 +80,35 @@ public void process() {
         renderFullscreenQuad();
         PerformanceMonitor.endActivity();
     }
+
+    public static OutputToScreenNode getInstance() {
+        return nodeInstance;
+    }
+
+    public void setFbo(String fboUri) {
```

Comment:
We are still handling the actual command inside the FBO, making every FBO responsible for it's own job.